### PR TITLE
add stats for firs, plis, nacks, packet counts

### PR DESCRIPTION
**Pull request by davibe**

### CODE REVIEW

File: str0m/src/media/sender.rs [hunk 1]
```diff
@@ -15,8 +16,15 @@ pub struct SenderSource {
     rid: Option<Rid>,
     next_seq_no: SeqNo,
     last_used: Instant,
-    bytes_tx: u64,
-    bytes_tx_resent: u64,
+    // bytes includes resent
+    bytes: u64,
+    bytes_resent: u64,
+    // packets includes resent
```
Comment (algesten) [inline str0m/src/media/sender.rs:25]: That `bytes` and `packets` also holds the resent is according to spec I assume?

Can we clarify the comment that this is according to spec? Maybe even link to the spec?

File: str0m/src/stats.rs [hunk 2]
```diff
@@ -97,6 +99,8 @@ pub struct MediaIngressStats {
     pub rid: Option<Rid>,
     // total bytes received
     pub bytes: u64,
+    // total number of rtp packets received
+    pub packets: u64,
```
Comment (algesten) [inline str0m/src/stats.rs:103]: This commit is a bit strange in isolation since it just adds struct fields without reading or writing to them.

**Review by algesten — APPROVED** [review]
LGTM! Two small comments

File: str0m/src/media/sender.rs (hunk 1 above)
Comment (davibe) [reply str0m/src/media/sender.rs:25]: correct, ok

Comment (davibe) [reply str0m/src/media/sender.rs:25]: I add the links to the stats events, not the sender/receiver because that is where the "public api" of the stats is

**Pull request merged**